.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -0,0 +1,38 @@
+name: 🐞 Bug Report
+description: Report a bug with MockBukkit.
+body:
+  - type: checkboxes
+    attributes:
+      label: Is there an existing issue for this?
+      description: Please search to see if an issue already exists for the bug you encountered.
+      options:
+        - label: I have searched the existing issues
+          required: true
+  - type: checkboxes
+    attributes:
+      label: Are you using the latest version of MockBukkit?
+      description: Please make sure you're using the latest version of MockBukkit as it's possible your issue has already been fixed.
+      options:
+        - label: I am using the latest version of MockBukkit.
+          required: true
```

Comment:
It would probably be a better idea to also ask for the minecraft version. For example have a drop down with minecraft versions